src/header.ce.vue

```diff
@@ -27,75 +37,231 @@ const loginUrl = computed(() => {
 })
 const logoutUrl = computed(() => '/logout')
 
+function toggleMenu(): void {
+  state.mobileMenuOpen = !state.mobileMenuOpen
+}
+
+const LANG_2_TO_3_MAPPER: { [index: string]: any } = {
+  en: 'eng',
+  nl: 'dut',
+  fr: 'fre',
+  de: 'ger',
+  ko: 'kor',
+  es: 'spa',
+  cs: 'cze',
+  ca: 'cat',
+  fi: 'fin',
+  is: 'ice',
+  it: 'ita',
+  pt: 'por',
+  ru: 'rus',
+  zh: 'chi',
+  sk: 'slo',
+}
+
 onMounted(() => {
+  state.lang3 =
+    LANG_2_TO_3_MAPPER[props.lang || navigator.language.substring(0, 2)] ||
+    'eng'
   getUserDetails().then(user => {
     state.user = user
   })
 })
 </script>
 <template>
-  <div v-if="props.legacyUrl">
+  <!-- props.legacyHeader -->
+  <div v-if="'true' == props.legacyHeader">
     <iframe
-      v-bind:src="props.legacyUrl"
-      v-bind:style="props.legacyStyle"
+      v-bind:src="
+        props.legacyUrl + (props.activeApp ? '?active=' + props.activeApp : '')
+      "
+      v-bind:style="props.style"
     ></iframe>
   </div>
-  <header v-if="!props.legacyUrl" class="host">
-    <div
-      class="admin pr-8 items-center bg-primary/20 text-secondary/80 flex justify-end gap-5 text-sm font-sans"
-      v-if="isAdmin"
-    >
-      <div class="py-1 bg-secondary/80 text-slate-100 px-8">Administration</div>
-      <a
-        href="/geonetwork/srv/fre/admin.console"
-        class="catalog py-1 hover:text-secondary/60"
-        v-if="adminRoles?.catalog"
-        >catalog</a
-      >
-      <a
-        href="/mapstore/#/admin"
-        v-if="adminRoles?.viewer"
-        class="py-1 hover:text-secondary/60"
-        >mapstore</a
-      >
-      <a
-        href="/console/manager/home"
-        v-if="adminRoles?.console"
-        class="console py-1 hover:text-secondary/60"
-        >console</a
-      >
-      <a href="/analytics/" class="console py-1 hover:text-secondary/60"
-        >analytics</a
-      >
-    </div>
-    <div class="h-20 flex justify-between text-slate-600">
+  <header v-else class="host h-full" v-bind:style="props.style">
+    <div class="justify-between text-slate-600 sm:flex hidden h-full">
```

Comment:
I think this got lost:
```suggestion
    <div class="justify-between bg-white text-slate-600 sm:flex hidden h-full">
```